elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
   title--m-sm--FontSize:    #{pfe-var(FontSize--sm ) },   // 14px
   title--m-xs--FontSize:    #{pfe-var(FontSize--xs ) },   // 12px
 );
+
+$pf-typography-sets: (
```

Comment:
This is the money-maker right here.  I'd like to ideally get this information out of a design token someday which would let us sprinkle the goodness in the library. cc @dcaryll @coreyvickery